google/cloud/security/inventory/inventory_loader.py

```diff
@@ -71,6 +73,8 @@
                     'The file with credentials for the service account.'
                     'NOTE: This is only required when running locally.')
 flags.DEFINE_string('organization_id', None, 'Organization ID.')
+flags.DEFINE_integer('max_crm_api_calls_per_100_seconds', 400,
```

Comment:
Yes, I think this flag is needed for when people requests quota increases and there is a way to configure the increased quota.  What might work better is to move the pipeline-specific flags to each of the pipeline or api modules.  Will look into that.